routes/web.php

```diff
@@ -19,8 +19,13 @@
     return view('category');
 });
 
+// Route::get('/user', function () {
```

Comment:
if you do not need this in this PR, please remove

Reply:
I don't need it, so I removed it.